README.md

```diff
@@ -37,22 +40,25 @@ We’re glad you’d like to use the Standards — here’s how you can get star
 
 * Designers: [Check out our Getting Started for Designers information](https://standards.usa.gov/getting-started/designers/).
     * [Design files of all the assets included in the Standards are available for download](https://github.com/18F/web-design-standards-assets/archive/master.zip).
-* Developers: Follow the instructions in this README to get started.
+* Developers: [Follow the instructions in this README to get started.](#using-the-standards)
     * [CSS, JavaScript, image, and font files of all the assets on this site are available for download](https://github.com/18F/web-design-standards/releases/download/v0.13.1/uswds-0.13.1.zip).
 
 ## Using the Standards
 
-There are a few different ways to use the Standards within your project. Which one you choose depends on the needs of your project and how you are most comfortable working.
+There are a few different ways to use the Standards within your project. Which one you choose depends on the needs of your project and how you are most comfortable working. Here are a few notes on what to consider when deciding which installation method to use:
 
-### Download
+*Download the Standards if:*
+- You are not familiar with `npm` and package management.
 
-To use the Draft Web Design Standards on your project, you’ll need to include the CSS and JavaScript files in each HTML page in your project.
+*Use the Standards `npm` package if:*
```

Comment:
@shawnbot :thinking: do you think that our users would understand that as the name of the package? I don't think we've outlined that anywhere else in our docs.